publisher-sdk/src/main/java/com/criteo/publisher/CriteoInterstitial.java

```diff
@@ -79,16 +79,23 @@ public CriteoInterstitial(
   }
 
   public void setCriteoInterstitialAdListener(
-      @Nullable CriteoInterstitialAdListener criteoInterstitialAdListener) {
+      @Nullable CriteoInterstitialAdListener criteoInterstitialAdListener
+  ) {
     this.criteoInterstitialAdListener = criteoInterstitialAdListener;
   }
 
   public void setCriteoInterstitialAdDisplayListener(
-      @Nullable CriteoInterstitialAdDisplayListener criteoInterstitialAdDisplayListener) {
+      @Nullable CriteoInterstitialAdDisplayListener criteoInterstitialAdDisplayListener
+  ) {
     this.criteoInterstitialAdDisplayListener = criteoInterstitialAdDisplayListener;
   }
 
   public void loadAd() {
+    if (!DependencyProvider.getInstance().isApplicationSet()) {
+      Log.w(TAG, "Calling CriteoInterstitial#loadAd with a null applicationContext");
```

Comment:
Maybe the message should be `Calling CriteoInterstitial#loadAd with a null application` to be consistent with other ones?